### PR TITLE
[vividus] Align wordings of file steps

### LINKED ISSUE #2042: Align steps wordings for temp files
Align steps wording to be:
```
When I create file with pathname `$pathname` and content `$fileContent`
When I create temporary file with name `$name` and content `$content` and put path to `$scopes` variable with name `$pathVariable`

```

_Originally posted by @ikalinin1 in https://github.com/vividus-framework/vividus/pull/2015#r721388816_

### PR DESCRIPTION
**Pull request by ikalinin1**
closes #2042

### CODE REVIEW

File: docs/modules/commons/pages/vividus-steps.adoc [hunk 1]
```diff
@@ -159,7 +159,7 @@ Saves the provided content to a file with the specified pathname.
 
 [source,gherkin]
 ----
-When I create a file with the pathname `$pathname` and the content `$fileContent`
+When I create file with pathname `$pathname` and content `$fileContent`
```
Comment (valfirst) [inline docs/modules/commons/pages/vividus-steps.adoc:162]:
```suggestion
When I create file with content `$fileContent` at path `$filePath`
```

File: docs/modules/commons/pages/vividus-steps.adoc [hunk 2]
```diff
@@ -173,7 +173,7 @@ The created file will be removed upon test run completion.
 
 [source,gherkin]
 ----
-When I create temporary file with name '$name' and content '$content' and put path to '$scopes' variable with name '$pathVariable'
+When I create temporary file with name `$name` and content `$content` and put path to `$scopes` variable `$pathVariable`
```
Comment (valfirst) [inline docs/modules/commons/pages/vividus-steps.adoc:176]:
```suggestion
When I create temporary file with name `$name` and content `$content` and put path to $scopes variable `$variableName`
```

File: vividus/src/main/resources/steps/defaults/temp-file.steps [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Composite: When I create temporary file with name '$name' and content '$content' and put path to '$scopes' variable with name '$pathVariable'
+!-- Step is deprecated, use: When I create temporary file with name `$name` and content `$content` and put path to `$scopes` variable `$pathVariable`
```
Comment (valfirst) [inline vividus/src/main/resources/steps/defaults/temp-file.steps:2]:
```suggestion
!-- WARNING: The step "When I create temporary file with name '$name' and content '$content' and put path to '$scopes' variable with name '$pathVariable'" is deprecated and will be removed in VIVIDUS 0.4.0
```

File: vividus/src/main/resources/steps/defaults/temp-file.steps [hunk 4]
```diff
@@ -0,0 +1,7 @@
...
+When I create temporary file with name `<name>` and content `<content>` and put path to `<scopes>` variable `<pathVariable>`
+
+Composite: When I create a file with the pathname `$pathname` and the content `$fileContent`
+!-- Step is deprecated, use: When I create file with pathname `$pathname` and content `$fileContent`
```
Comment (valfirst) [inline vividus/src/main/resources/steps/defaults/temp-file.steps:6]:
```suggestion
!-- WARNING: The step "When I create a file with the pathname `$pathname` and the content `$fileContent`" is deprecated and will be removed in VIVIDUS 0.4.0
```

**Review by valfirst — CHANGES_REQUESTED** [review]

File: vividus/src/main/resources/steps/defaults/temp-file.steps [hunk 5]
```diff
@@ -0,0 +1,7 @@
+Composite: When I create temporary file with name '$name' and content '$content' and put path to '$scopes' variable with name '$pathVariable'
```
Comment (ikalinin1) [inline vividus/src/main/resources/steps/defaults/temp-file.steps:1]: Remove deprecated steps

**Review by abudevich — APPROVED** [review]

File: docs/modules/commons/pages/vividus-steps.adoc [hunk 6]
```diff
@@ -159,11 +159,11 @@ Saves the provided content to a file with the specified pathname.
 
 [source,gherkin]
 ----
-When I create a file with the pathname `$pathname` and the content `$fileContent`
+When I create file with content `$fileContent` at path `$pathname`
```
Comment (valfirst) [inline docs/modules/commons/pages/vividus-steps.adoc:162]: users don't understand what `pathname` is, that's too technical for them
```suggestion
When I create file with content `$fileContent` at path `$filePath`
```

File: docs/modules/commons/pages/vividus-steps.adoc [hunk 7]
```diff
@@ -173,7 +173,7 @@ The created file will be removed upon test run completion.
 
 [source,gherkin]
 ----
-When I create temporary file with name '$name' and content '$content' and put path to '$scopes' variable with name '$pathVariable'
+When I create temporary file with name `$name` and content `$content` and put path to $scopes variable `$pathVariable`
```
Comment (valfirst) [inline docs/modules/commons/pages/vividus-steps.adoc:176]: we need to define explicitly that it's a name of the variable, not the variable reference
```suggestion
When I create temporary file with name `$name` and content `$content` and put path to $scopes variable `$variableName`
```

Comment (valfirst) [reply docs/modules/commons/pages/vividus-steps.adoc:176]: btw the step syntax was updated, but here it's not

File: vividus/src/main/java/org/vividus/bdd/steps/FileSteps.java [hunk 8]
```diff
@@ -48,27 +48,27 @@
...
      * Saves content to file with specified pathname
-     * @param pathname Fully qualified file name with parent folders and extension (e.g. temp/some_file.txt)
      * @param fileContent Content to be saved to file
+     * @param pathname Fully qualified file name with parent folders and extension (e.g. temp/some_file.txt)
      * @throws IOException If an I/O error has occurred
      */
-    @When("I create a file with the pathname `$pathname` and the content `$fileContent`")
-    public void createFile(String pathname, String fileContent) throws IOException
+    @When("I create file with content `$fileContent` at path `$filePath`")
+    public void createFile(String fileContent, String pathname) throws IOException
```
Comment (valfirst) [inline vividus/src/main/java/org/vividus/bdd/steps/FileSteps.java:71]:
```suggestion
    public void createFile(String fileContent, String filePath) throws IOException
```

**Review by valfirst — CHANGES_REQUESTED** [review]

**Review by valfirst — APPROVED** [review]

**Review by uarlouski — APPROVED** [review]

**Pull request merged**